### PR TITLE
12.0 pypi first packaging

### PR DESCRIPTION
**Pull request by raneq**
Packaged and (already) uploaded modules:

- [x] [easy_my_coop](https://pypi.org/project/odoo12-addon-easy-my-coop/)
- [x] [easy_my_coop_be](https://pypi.org/project/odoo12-addon-easy-my-coop-be/12.0.1.2.0.99.dev4/)
- [x] [easy_my_coop_ch](https://pypi.org/project/odoo12-addon-easy-my-coop-ch/12.0.1.0.2.99.dev5/)
- [ ] ~easy_my_coop_dividend~ (requires odoo9 and python 2.7)
- [ ] ~easy_my_coop_document~ (requires odoo9 and python 2.7)
- [x] [easy_my_coop_eater](https://pypi.org/project/odoo12-addon-easy-my-coop-eater/)
- [ ] ~easy_my_coop_export_xlsx~ (requires odoo9 and python 2.7)
- [x] [easy_my_coop_fr](https://pypi.org/project/odoo12-addon-easy-my-coop-fr/12.0.1.0.1.99.dev6/)
- [x] [easy_my_coop_loan](https://pypi.org/project/odoo12-addon-easy-my-coop-loan/12.0.1.0.1.99.dev40/)
- [x] [easy_my_coop_loan_website](https://pypi.org/project/odoo12-addon-easy-my-coop-loan-website/12.0.1.0.1.99.dev1/)
- [ ] ~easy_my_coop_online_payment~ (requires odoo9 and python 2.7)
- [x] [easy_my_coop_recompute_coop_number](https://pypi.org/project/odoo12-addon-easy-my-coop-recompute-coop-number/12.0.1.0.0.99.dev1/)
- [ ] ~easy_my_coop_taxshelter_report~ (requires odoo9 and python 2.7)
- [x] [easy_my_coop_website](https://pypi.org/project/odoo12-addon-easy-my-coop-website/)
- [ ] ~easy_my_coop_website_document~ ( [Migration still WIP](https://github.com/coopiteasy/vertical-cooperative/pull/40) enforced by `'installable: False'`)
- [x] easy_my_coop_website_portal ~Not migrated. Locked by `'installable: False'`~ Fixed at 865d6cff4b6fe8d25ec5df779c1d2691845a5d27
- [ ] ~easy_my_coop_website_taxshelter~ Not migrated. Locked by `'installable: False'`
- [x] [partner_age](https://pypi.org/project/odoo12-addon-partner-age/12.0.1.0.0.99.dev1/)
- [x] [theme_light](https://pypi.org/project/odoo12-addon-theme-light/12.0.1.0.0.99.dev1/)
- [ ] ~website_portal_extend~ (requires odoo9 and python 2.7)
- [x] [website_recaptcha_reloaded](https://pypi.org/project/odoo12-addon-website-recaptcha-reloaded/)

This smells like intercooperation :grin:

### CODE REVIEW

**Comment by raneq** [comment]
Trying to make the process smoother, I rebased my local commits, merged the upstream branch 12.0 onto mine and renamed my branch with 12.0 prefix, following your style.

I hope it suits you! However, please feel free to request any changes.

**Comment by enricostano** [comment]
@remytms @houssine78  hi! How do you feel about merging this PR? Is there anything that we could do to help? Thanks!

**Comment by houssine78** [comment]
@enricostano I'll let @remytms have a look on this.

**Comment by enricostano** [comment]
Hi @houssine78 @remytms do you have any news about this? Thanks!

**Comment by houssine78** [comment]
Hi @enricostano I think that you need to rebase and resolve conflict. 

Juste one remark, Do the website modification is really necessary why removing the www ?

**Review by remytms — APPROVED** [review]
Ok for me. I understand the removal of `www`. If I remember well, the website is not considered as a link if it does not begins with `http` or `https`.

I will merge this as soon as possible. :)

File: easy_my_coop_online_payment/__manifest__.py [hunk 1]
```diff
@@ -20,13 +20,13 @@
 ##############################################################################
 {
     "name": "Easy My Coop Online Payment",
-    "version": "1.0",
+    "version": "9.0.1.0.0",
```
Comment (enricostano) [inline easy_my_coop_online_payment/__manifest__.py:23]: @remytms should this version start with `12`?

File: easy_my_coop_export_xlsx/__manifest__.py [hunk 2]
```diff
@@ -27,8 +27,8 @@
     """,
     'author': 'Houssine BAKKALI, <houssine@coopiteasy.be>',
     'license': 'AGPL-3',
-    'version': '9.0.1.0',
-    'website': "www.coopiteasy.be",
+    'version': '9.0.1.0.0',
```
Comment (enricostano) [inline easy_my_coop_export_xlsx/__manifest__.py:30]: @remytms should this version start with `12`?

File: easy_my_coop_dividend/__manifest__.py [hunk 3]
```diff
@@ -33,8 +33,8 @@
 
     'author': 'Houssine BAKKALI, <houssine@coopiteasy.be>',
     'license': 'AGPL-3',
-    'version': '9.0.1.0',
-    'website': "www.coopiteasy.be",
+    'version': '9.0.1.0.0',
```
Comment (enricostano) [inline easy_my_coop_dividend/__manifest__.py:36]: @remytms should this version start with `12`?

**Comment by enricostano** [comment]
@remytms @houssine78 rebased. I just have a couple of comments for some versions (see above in the comments).

Many thanks! This will improve a lot our processes :muscle:

**Comment by remytms** [comment]
@enricostano These modules has the 9.0 version tag, because there are not yet migrated to 12.0. It should contain the tag `"installable": False`. It's an old process from OCA, to mark module as not installable when there are not migrated yet.

If the tag `"installable": False` is not present in the manifest of these module, I should be added.

@houssine78 Do you agree with my analysis ?

**Comment by houssine78** [comment]
@remytms normally it the module hasn't been migrated it stays with the old manifest name __openerp__.py it won't be recognized as an installable module.

for the installable flag. I think but I'm not sure that it's False by default but explicit is better than implicit ! :)

File: easy_my_coop_dividend/__manifest__.py (hunk 3 above)
Comment (houssine78) [reply easy_my_coop_dividend/__manifest__.py:36]: yes it should.

File: easy_my_coop_online_payment/__manifest__.py (hunk 1 above)
Comment (houssine78) [reply easy_my_coop_online_payment/__manifest__.py:23]: this one shouldn't be migrated as it is. we should remove it from the 12 branch as it should be completely refactored and get ride of paypal.

@remytms could you just throw it away from 12.0 branch ?

**Comment by enricostano** [comment]
Thanks @houssine78 @remytms !

Anything that I could do to move this forward?

**Comment by houssine78** [comment]
Hi @enricostano @remytms will take care of this PR.

File: easy_my_coop_online_payment/__manifest__.py (hunk 1 above)
Comment (remytms) [reply easy_my_coop_online_payment/__manifest__.py:23]: Done.

File: easy_my_coop_export_xlsx/__manifest__.py (hunk 2 above)
Comment (remytms) [reply easy_my_coop_export_xlsx/__manifest__.py:30]: @enricostano You're right. :) It's done.

File: easy_my_coop_dividend/__manifest__.py (hunk 3 above)
Comment (remytms) [reply easy_my_coop_dividend/__manifest__.py:36]: Done.

**Comment by remytms** [comment]
We will first merge this PR #84 .

**Comment by enricostano** [comment]
@houssine78 @remytms @robinkeunen now that #84 has been merged we need to rebase this PR. Do you need any help from our side? Thanks!

**Comment by houssine78** [comment]
Hi @enricostano I think that this should comes from @raneq

**Comment by enricostano** [comment]
@houssine78 ok, don't worry with @cesarlr we'll rebase it :muscle: Thanks!

**Comment by enricostano** [comment]
Superseded by https://github.com/coopiteasy/vertical-cooperative/pull/104
We can close this one @houssine78

**Pull request closed without merging**